testapi.pm

```diff
@@ -161,7 +158,7 @@ You can use distribution object to implement distribution specific helpers.
 
 =cut
 
-sub set_distribution {
+sub set_distribution {    # no:style:signatures
```

Comment:
Why can't this be converted? Looks straight forward. 🤔

Reply:
I got weird errors that I did not understand converting this. But it's easy to try, just change to signatures and run `prove -I. t/03-testapi.t` and you should see the error – or not when you are doing it better than I could do :)